src/staging/staging_main.py

```diff
@@ -216,7 +216,11 @@ def run_staging(
             os.path.join(feather_path, f"{snapshot_name}_responses.feather")
         )
         full_responses = snapdata
-        secondary_full_responses = secondary_snapdata
+
+        if load_updated_snapshot:
```

Comment:
I don't know about this change and although the flag here is descriptive, I still don't know what an "updated snapshot" is, so perhaps a comment is needed here.

Reply:
in the interests of getting this code in the release, could the comments relating to other branches (Tom's construction code was breaknig something)  be left for later?